_posts/2019-11-04-functional-traits.md

```diff
@@ -47,14 +47,18 @@ The most common metric used in trait-based studies is the community weighted mea
 
 <strong> </strong>
 <h4><strong>4.16.2 Special cases, emerging issues, and challenges</strong></h4>
+All ecosystems have different challenges, and hence the importance of different traits vary between ecosystems. Open ecosystems such as grasslands, savannas, open woodlands and shrublands are often exposed to disturbances such as herbivory, fires and drought. To understand how climate change affects these systems, several traits that were not present in the Pérez-Harguindeguy et al. (2013) handbook were useful, and Wigley et al. (2021) gather tham all in a handbook for trait measurements in open ecosystems.  
+
 Currently, existing trait protocols and databases for plants do not include cryptogams. Nonetheless, there have been advances in developing trait protocols for cryptogams such as bryophytes and lichens (Cornelissen et al., 2007; St. Martin &amp; Mallik, 2017) and soil crusts (Mallen-Cooper &amp; Eldridge, 2016). While no large database has been developed, there have been renewed calls for the integration of cryptogram trait ecology into plant-based trait studies (Deane‐Coe &amp; Stanton, 2017; St. Martin &amp; Mallik, 2017).
 
 Trait-based ecology has proliferated in many non-plant taxa as well, offering substantial value to climate-change studies. However, given the increased trait specialisation across and within the other organismal domains, we do not cover any other taxa in detail here. Considerations of response and effect traits and inter- and intra-specific variability also apply to non-plant taxa. The availability of trait databases is highly variable across guilds of species and include the PanTHERIA database for mammals (Jones et al., 2009), an amniote database for birds, reptiles, and mammals (Myhrvold et al., 2015), the GlobalAnts database that includes both abundance and trait data linked to local assemblages (Parr et al., 2017), and the FUNGuild database which has begun classifying fungal operational taxonomic units identified by high throughput sequencing into functional guilds (Nguyen et al., 2016). This is not a comprehensive list of trait databases (many databases exist for aquatic organisms), and regional scientific societies often curate their own such databases for a variety of organisms.
 
+Several studies have showed that traits vary through ontogeny (Gibert et al. 2016; Damián et al. 2018), and that seedling traits can be important in climate change studies linked to for example drought and warming (Harrison & LaForgia 2019). Using functional traits in the early life stages, and through ontogeny, is an emerging topic that could help give better predictions for effects of climate change on plant communities and ecosystems. Most of these traits are measured in the same way as for adults, following the same protocol (Pérez-Harguindeguy et al. 2013), althought the small size might make that difficult for some species at the earliest life stages.
```

Comment:
Several studies have shown???

"...is an emerging topic that could help give better predictions for effects of climate change on plant communities and ecosystems."
Yes this is true but also a bit general. Looking at the early stages will give a better understanding of how species establish etc. can we write this sentence a bit more specific in this sense?